### PR TITLE
Removes unnecessary graphics-drivers ppa repository

### PR DESCRIPTION
**Pull request by perdasilva**
## Description ##
Removes superfluous ppa repository add in ci script that installs nvidia dependencies

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

**Comment by perdasilva** [comment]
@mxnet-label-bot update [pr-awaiting-review]

File: ci/docker/install/ubuntu_nvidia.sh [hunk 1]
```diff
@@ -18,11 +18,6 @@
 # under the License.
 
 set -ex
-apt-get update || true
-apt install -y software-properties-common
-
-# Adding ppas frequently fails due to busy gpg servers, retry 5 times with 5 minute delays.
-for i in 1 2 3 4 5; do add-apt-repository -y ppa:graphics-drivers && break || sleep 300; done
 
 # Retrieve ppa:graphics-drivers and install nvidia-drivers.
```
[the remark this answers is not in the document]
Comment (perdasilva) [reply ci/docker/install/ubuntu_nvidia.sh]: I believe this installs the nvidia drivers and libcuda.so - which is needed by mxnet

**Comment by perdasilva** [comment]
@mxnet-label-bot update [pr-awaiting-merge]

File: ci/docker/install/ubuntu_nvidia.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (perdasilva) [reply ci/docker/install/ubuntu_nvidia.sh]: @lebeg - I did a little more digging: https://github.com/apache/incubator-mxnet/blob/master/ci/docker/Dockerfile.build.ubuntu_build_cuda#L47

[the remark this answers is not in the document]
Comment (lebeg) [reply ci/docker/install/ubuntu_nvidia.sh]: Oh, I see. Thanks!

**Pull request merged**